cpp/msbuild/ice.nuget.targets

```diff
@@ -14,36 +14,36 @@
-                                  $(IceSrcRootDir)lib\$(Platform)\$(Configuration)\icexml*.lib;
                                   $(IceSrcRootDir)lib\$(Platform)\$(Configuration)\slice*.lib"/>
     </ItemGroup>
 
-    <!-- Executables & Tools -->
-    <ItemGroup>
-
-        <ExecutableExcludes Include="$(IceSrcRootDir)bin\$(Platform)\$(Configuration)\glacier2router.exe;
-                                     $(IceSrcRootDir)bin\$(Platform)\$(Configuration)\iceboxadmin.exe;
-                                     $(IceSrcRootDir)bin\$(Platform)\$(Configuration)\icebridge.exe;
-                                     $(IceSrcRootDir)bin\$(Platform)\$(Configuration)\icegridadmin.exe;
-                                     $(IceSrcRootDir)bin\$(Platform)\$(Configuration)\icegriddb.exe;
-                                     $(IceSrcRootDir)bin\$(Platform)\$(Configuration)\iceserviceinstall.exe;
-                                     $(IceSrcRootDir)bin\$(Platform)\$(Configuration)\icestormadmin.exe;
-                                     $(IceSrcRootDir)bin\$(Platform)\$(Configuration)\icestormdb.exe"
-                            Condition="'$(DefaultBuild)' != '$(Platform)|$(Configuration)' and '$(Configuration)' == 'Debug'"/>
-
-        <ExecutableExcludes Include="$(IceSrcRootDir)bin\$(Platform)\$(Configuration)\slice2*.exe"/>
-
-        <Executables Include="$(IceSrcRootDir)bin\$(Platform)\$(Configuration)\*.exe"
-                     Exclude="@(ExecutableExcludes)"/>
+    <ItemGroup Condition="'$(DefaultBuild)' != '$(Platform)|$(Configuration)' and '$(Configuration)' == 'Debug'">
+        <!--The below executables are only included for the default platform -->
+        <ExecutablesExclude Include="$(IceSrcRootDir)bin\$(Platform)\$(Configuration)\dsnode.exe" />
+        <ExecutablesExclude Include="$(IceSrcRootDir)bin\$(Platform)\$(Configuration)\glacier2router.exe" />
+        <ExecutablesExclude Include="$(IceSrcRootDir)bin\$(Platform)\$(Configuration)\iceboxadmin.exe" />
+        <ExecutablesExclude Include="$(IceSrcRootDir)bin\$(Platform)\$(Configuration)\icebridge.exe" />
+        <ExecutablesExclude Include="$(IceSrcRootDir)bin\$(Platform)\$(Configuration)\icegridadmin.exe" />
+        <ExecutablesExclude Include="$(IceSrcRootDir)bin\$(Platform)\$(Configuration)\icegriddb.exe" />
+        <ExecutablesExclude Include="$(IceSrcRootDir)bin\$(Platform)\$(Configuration)\iceserviceinstall.exe" />
+        <ExecutablesExclude Include="$(IceSrcRootDir)bin\$(Platform)\$(Configuration)\icestormadmin.exe" />
+        <ExecutablesExclude Include="$(IceSrcRootDir)bin\$(Platform)\$(Configuration)\icestormdb.exe" />
+    </ItemGroup>
 
+    <ItemGroup>
+        <!-- The Slice compilers are includes as Tools -->
         <Tools Include="$(IceSrcRootDir)bin\$(Platform)\$(Configuration)\slice2*.exe" />
+        <Tools Include="$(IceSrcRootDir)bin\$(Platform)\$(Configuration)\ice2slice.exe" />
```

Comment:
ice2slice belongs to the tools directory, it was included in bin before.

Reply:
Presumably all tools are linked statically?

Reply:
yes